bin/pygrb/pycbc_pygrb_page_tables

```diff
@@ -616,18 +457,11 @@ else:
 # Post-process injections
 # =======================
 if do_injections:
-
-    found_trig_table = ppu.load_injections(found_missed_file, vetoes)
-    found_inj_table = ppu.load_injections(found_missed_file, vetoes, sim_table=True)
-
+    found_injs, missed_injs = \
+        ppu.load_missed_found_injections(found_missed_file, ifos, opts,
+                                background_bestnrs=full_time_veto_max_bestnr)
```

Comment:
Get these using the `FOUNDMISSED` injection file with `ppu.read_triggers`.

Reply:
Do you mean`load_triggers`?